sonar-project.properties

```diff
@@ -6,5 +6,5 @@ sonar.sources=src/main
 sonar.tests=src/test
 sonar.test.exclusions=test/coverage/**
 sonar.javascript.lcov.reportPaths=coverage/lcov.info
-sonar.exclusions=src/main/app.ts,src/main/development.ts,src/main/index.js,src/main/server.ts,src/main/modules/i18next/index.ts,src/main/bundles/cookie-preferences.ts,src/main/modules/helmet/index.ts
+sonar.exclusions=src/main/app.ts,src/main/development.ts,src/main/index.js,src/main/server.ts,src/main/bundles/cookie-preferences.ts,src/main/modules/**/*.ts
```

Comment:
I'd avoid the wildcard exclusion here. Having them defined individually means any modules added in the future will correctly fail unless explicitly excluded

Reply:
This will also be excluded in my PR for ... 1509? I think